src/main/java/codeu/model/data/User.java

```diff
@@ -23,20 +23,23 @@
   private final String name;
   private final String passwordHash;
   private final Instant creation;
+  private String bio;
 
   /**
-   * Constructs a new User.
+   * Constructs a new User & initially sets an empty bio for user.
    *
    * @param id the ID of this User
    * @param name the username of this User
    * @param passwordHash the password hash of this User
    * @param creation the creation time of this User
+   * @param bio the bio of this User
    */
-  public User(UUID id, String name, String passwordHash, Instant creation) {
+  public User(UUID id, String name, String passwordHash, Instant creation, String bio) {
     this.id = id;
     this.name = name;
     this.passwordHash = passwordHash;
     this.creation = creation;
+    this.bio = bio;
```

Comment:
I was wondering if you could handle setting up a new user's initial bio here:
 this.bio = "This user hasn't create a bio yet";

Reply:
ooh this is a good idea, and might be better b/c all Users created have the same bio anyways.. plus then I won't have to fix the Unit Tests that already work. I'll try this!